.github/workflows/release-pr.yml

```diff
@@ -70,12 +70,6 @@ jobs:
           git add -A
           git commit -m "chore: post-release go.mod updates"
 
-          # TODO: https://github.com/pulumi/pulumi/issues/15458
-          # Temporarily disable tagging `pkg`. For now, we'll have to manually do it
-          # once the change lands in `master`.
-          # # Publish pkg module on another tag.
-          # git tag "pkg/v${PULUMI_VERSION}"
-          # git push origin "pkg/v${PULUMI_VERSION}"
           git push -u origin HEAD
 
           PR=$(gh pr create --title "Changelog and go.mod updates for v${PULUMI_VERSION}" --body "")
```

Comment:
If we're using the title to determine it needs the tag, we should probably add a comment here mentioning that so we don't unknowingly change the title in the future and break the automation.